src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -2406,4 +2410,26 @@ public static void closeErrorDialog(RemoteRobot remoteRobot) {
             okButton.click();
         }
     }
+
+    /**
+     * Cleans up resources and stops the Liberty server if necessary.
+     *
+     * @param remoteRobot       The RemoteRobot instance.
+     * @param absoluteWLPPath   The absolute path of the Liberty installation.
+     * @param projectName       The name of the project to select.
+     * @param testName          The name of the test calling this method.
+     */
+    public static void cleanupAndStopServerIfNeeded(RemoteRobot remoteRobot, String absoluteWLPPath, String projectName, String testName) {
+
+        // Open the terminal window.
+        openTerminalWindow(remoteRobot);
+
+        if (TestUtils.isServerStopNeeded(absoluteWLPPath)) {
+            // Sleep for a few seconds to allow dev mode to finish running the tests.
+            TestUtils.sleepAndIgnoreException(60);
```

Comment:
dev mode is not running any tests in this scenario, so there is no need for the sleep.